src/web/resources/report.js

```diff
@@ -81,11 +81,10 @@ const Results = new Component("#results", {
         const regressedLeader = this.attachLeaderToChildren("regressed", "Regressed", regressedChildren)
 
         this.elt.parentNode.insertBefore(Element("div", { id: "filters" }, [
```

Comment:
I don't think you need the outer div here. Just make the `#filters` element the `details`.